tests/xtro-sharpie/api-annotations-dotnet/common-Foundation.ignore

```diff
@@ -790,7 +790,6 @@
 !unknown-type! NSObject bound
 
 ## Used 'INSFastEnumeration' placeholer (just so we can start flagging things)
-!missing-protocol-conformance! NSDictionary should conform to NSFastEnumeration (defined in 'NSGenericFastEnumeraiton' category)
```

Comment:
This has been removed here and added to each platform-specific file, which is not necessary.

Reply:
That was the tool, we might need to fix it.